website/docs/installation/ferretdb/deb.md

```diff
@@ -4,10 +4,21 @@ sidebar_position: 2
 
 # DEB package
 
+We provide different `.deb` packages for various deployments:
+
+- **a development package**: for debugging problems, and includes features that make it significantly slower.
```

Comment:
That's not a proper sentence, but a mishmash of existing documentation and a part copied from a Docker page